app/views/applications/show.html.erb

```diff
@@ -0,0 +1,42 @@
+<h1><%= @application.name + "'s Application" %></h1>
+<p>Name: <%= @application.name %></p>
+<p>Street Address: <%= @application.street_address %></p>
```

Comment:
This could probably get changed to be more appealing looking, rather than my weird bullet point version, haha.